packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketOfferSell/CoinmarketOfferSellTransaction.tsx

```diff
@@ -59,17 +59,17 @@ export const CoinmarketSelectedOfferSellTransaction = () => {
         account,
         trade,
     });
-    const t = trade?.data || selectedQuote;
+    const sellTrade = trade?.data || selectedQuote;
```

Comment:
😍

Reply:
If you have refactored this part, could you please refactor `WaitingWrapper` as in _CoinmarketOfferExchangeSend.tsx_? I know it is not part of this PR, but it would be awesome to have it. 🥳

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/15612/commits/28aa0b2cd126b163f2afa9199c800da73f474dd9